src/main/webapp/app/home/home.component.html

```diff
@@ -72,24 +72,33 @@ <h1 jhiTranslate="home.title">Welcome to Artemis!</h1>
                                 </label>
                             </div>
                         </div>
-                        <button
-                            type="submit"
-                            [disabled]="isSubmittingLogin || !acceptTerms || !password || password.length < 4 || !username || username.length < 4"
-                            class="btn btn-primary"
-                        >
-                            <span *ngIf="isSubmittingLogin" class="mr-1"><fa-icon [icon]="'circle-notch'" spin="true"></fa-icon></span>
-                            <span jhiTranslate="login.form.button"> Sign in </span>
-                        </button>
+                        <div class="btn-toolbar">
+                            <button
+                                type="submit"
+                                [disabled]="isSubmittingLogin || !acceptTerms || !password || password.length < 4 || !username || username.length < 4"
+                                class="btn btn-primary btn-space"
+                            >
+                                <span *ngIf="isSubmittingLogin" class="mr-1"><fa-icon [icon]="'circle-notch'" spin="true"></fa-icon></span>
+                                <span jhiTranslate="login.form.button"> Sign in </span>
+                            </button>
+                            <div *ngIf="saml2Enabled">
```

Comment:
would it make sense to extract this into a separate component?

Reply:
See https://github.com/ls1intum/Artemis/pull/3002#discussion_r602551008